components/Chips/src/MDCChipView.m

```diff
@@ -250,6 +252,16 @@ - (void)dealloc {
                                                 object:nil];
 }
 
+- (void)setShapeGenerator:(id)shapeGenerator {
```

Comment:
Did the type intentionally get dropped here?

Reply:
Fixed.